tests/framework/ROM/TimeSeries/ARMA/arma_reseed_test.xml

```diff
@@ -6,36 +6,31 @@
     <created>2017-07-12</created>
     <classesTested>SupervisedLearning.ARMA</classesTested>
     <description>
-      Runs an unpickled ARMA ROM in internal parallel and assures the histories
-      generated are not identical (expected_fail = True).  Note that the gold file
-      is created by coping ARMAparallel/results_1.csv to gold/ARMAparallel/results_0.csv,
-      then we make sure they are not the same through expected_fail
+      Runs an unpickled ARMA ROM in internal parallel and assures the histories generated are
```

Comment:
Except we changed this to be NOT internal parallel, right? We can just remove "in internal parallel" here.